test/v2/controllers/V2MovementsControllerSpec.scala

```diff
@@ -3364,8 +3364,11 @@ class V2MovementsControllerSpec
           }
       }
 
-      "when no movement is found" in forAll(arbitraryMovementId.arbitrary, arbitraryPageNumber.arbitrary, arbitraryItemCount.arbitrary) {
-        (movementId, pageNumber, itemCount) =>
+      // TODO: should this be MovementNotFound or PageNotFound ?
```

Comment:
This is the message list one, right?

* If the **movement** is not found, this is `MovementNotFound`. It forms part of the path, so will return 404.
* If the movement is found, but there are no messages, that's `PageNotFound`.

Reply:
Yep, agree - my misunderstanding.